src/modules/missionHelper/i18n/nb_NO.json

```diff
@@ -0,0 +1,93 @@
+{
+	"diyMission": "Dette oppdraget ser ut til å være en alliansehendelse i stor skala!",
+	"expansions": "Utvikling | Utviklinger",
+	"followups": "Follow-up Mission | Follow-up Missions",
+	"generated_by": "Generert av",
+	"noVehicleRequirements": [
+		"water_needed"
+	],
+	"patients": {
+		"allow_ktw_instead_of_rtw": "En syketransport kan bli brukt.",
+		"helicopter": "Sannsynlighet for Luftambulanse:",
+		"patient_allow_first_responder_chance": "Sannsynlighet for First Responder:",
+		"patient_at_end_of_mission": "NB: Pasienten dukker opp på slutten av oppdraget!",
+		"patient_transport": "Sannsynlighet for transport:",
+		"possible_patient": "Maksimalt antall pasienter",
+		"possible_patient_exact": "kun {n} pasient | kun {n} pasienter",
+		"possible_patient_min": "minimum pasienter",
+		"specializations": "Utdanning/Spesialitet",
+		"title": "Pasienter | {n} pasient | {n} Pasienter"
+	},
+	"prerequisites": {
+		"brush_extension": "Forestry Expansion | Forestry Expansions",
+		"federalpolice_stations": "Federal Police Station | Federal Police Stations",
+		"fire_aviation_count": "Firefighting Plane Station | Firefighting Plane Stations",
+		"fire_stations": "Fire Station | Fire Stations",
+		"max_police_stations": "Maximum Police Stations",
+		"max_rescue_stations": "Maximum Rescue Stations",
+		"police_helicopter_stations": "Police Helicopter Station | Police Helicopter Stations",
+		"police_stations": "Police Station | Police Stations",
+		"rescue_stations": "Ambulance Station | Ambulance Stations",
+		"title": "Prerequisite | Prerequisites"
+	},
+	"prisoners": {
+		"max_possible_prisoners": "Maksimalt antall fanger",
+		"title": "Fange | {n} fange | {n} fanger"
+	},
+	"requirements": {
+		"water_needed": "Water demand (gal)"
+	},
+	"vehicles": {
+		"captions": {
```

Comment:
![afbeelding](https://user-images.githubusercontent.com/65117490/98439875-2254ef00-20f5-11eb-8cd8-1a5d4cfe33fc.png)
Only need the above categories

Reply:
And add **trafic_car_only_if_needed** for the only if present option. Maybe Sanni or Jan could make the settings for that when  you don't know how to do in the settingfiles and root file.